ui/documents/semantic-ui/templates/documents/form/FormFieldsContainer.jsx

```diff
@@ -320,6 +332,11 @@ const FormFieldsContainer = () => {
               textFieldLabel={i18next.t("Description")}
               fieldPath="metadata.abstract"
               rich={true}
+              editorConfig={{
+                toolbar:
+                  "bold italic | bullist numlist | outdent indent | undo redo",
+                valid_elements: "strong,b,div,br,p,i,li",
```

Comment:
Inconsistent with https://github.com/oarepo/oarepo-ui/pull/184

Reply:
List of valid tags should come from a single source of truth, do not copy it in every possible input